Footer

Sidopjescherm opened the pull request:
## What does this change?
 
 
Een footer gemaakt voor de homepage die we later omzetten naar een component. Wat ik heb gemaakt is de footer met daarin de logo's, een stattisch navigatie menu en een contact info.
 
## Getest
Ik heb getest op:

- [x] Tab test
- [x] Focus test

## Images
### Desktop
<br>
<img width="1895" height="662" alt="desktop-full" src="https://github.com/user-attachments/assets/b79d0db5-391d-497d-bfaf-89b5fab30c0f" />
<br>
<img width="569" height="168" alt="desktop-logo's" src="https://github.com/user-attachments/assets/1b049d3b-1a76-468a-9e13-9f7944429df6" />
<br>
<img width="224" height="348" alt="desktop-nav" src="https://github.com/user-attachments/assets/be3b6712-258b-4894-85dd-27e9ce8f8b9e" />
<br>

<img width="334" height="517" alt="desktop-address" src="https://github.com/user-attachments/assets/f7e26c48-7bfd-4d10-8bd9-339de35a6c42" />

### Mobile
<br>
<img width="409" height="678" alt="Screenshot 2025-10-22 153901" src="https://github.com/user-attachments/assets/81bfd346-e2ef-4a8a-91f1-84b26d09137b" />
<br>
<img width="395" height="236" alt="Screenshot 2025-10-22 153908" src="https://github.com/user-attachments/assets/ca53fd8b-af10-4fdd-9a4c-b0276fab45e7" />
<br>
<img width="219" height="57" alt="Screenshot 2025-10-22 154710" src="https://github.com/user-attachments/assets/b580c718-3235-4cec-8654-9d10b1f07178" />
<br>
<img width="127" height="74" alt="Screenshot 2025-10-22 154723" src="https://github.com/user-attachments/assets/84bd83cd-a095-4cf2-94de-3196009462d8" />
<br>
 
## Waarop testen?
- Tab test
- Focus test
- Screen reader
- Code (voor DRY en PE)

Sidopjescherm committed:
#275 Opzet footer mobile

In deze commit heb ik de eerste opzet naar de mobiel versie. Gelukkig kan ik de meeste code van @nadiachaja overnemen vanuit de header.

Ik heb het Address element gebruikt en alles een logische plek gegeven zoals het in het Figma document staat.

@nadiachaja wat vind jij van dit tot nu toe?

Sidopjescherm committed:
#275 Fix navigatie

Omdat ik problemen tegenkwam tijdens het werken aan de footer met navigatie had ik eerst er een article van gemaakt. Éénmaal na het uivogelen van de navigatie uit de header heb ik het aangepast zodat het in de footer past.

Sidopjescherm committed:
#275 Fix address

Nu heeft het address met alles dat erin zit de juiste font-style en family

Sidopjescherm committed:
#275 Grid footer

Grid van de footer aangepast voor kleine schermen en grote schermen

nadiachaja committed:
#308 hebt classname op nav header gezet

Sidopjescherm committed:
#275 Footer voor desktop en mobile

Ik heb de basic footer afgemaakt voor grote schermen en kleine schermen. Ook werkt de tab functie met een duidelijke :focus state zodat je ziet waar je bent.

Sidopjescherm committed:
#275 Alt weggehaald

nadiachaja commented:
Ik zou de logo's niet mee laten bewegen zou de logo's de mobile versie en desktop versie gebruiken. 
Ik de footer veel ruimte hebben met veel witruimte.
<img width="674" height="378" alt="Scherm­afbeelding 2025-10-22 om 16 47 22" src="https://github.com/user-attachments/assets/14e6bbde-2f02-48b3-843f-bee15c2c2327" />

Zou misschien bij tablet de navigatie en contactinformatie naast elkaar zetten. 
<img width="493" height="522" alt="Scherm­afbeelding 2025-10-22 om 16 58 39" src="https://github.com/user-attachments/assets/17ec3842-e926-448d-8209-fe7b05160551" />

Mobile ziet het er goed uit.

Sidopjescherm committed:
#309 Footer fix

Ik heb de images vervangen door dezelfde die jij ook gebruikt @nadiachaja en het werkt wel beter. Ik schaal de afbeeldingen correct met een clamp zodat ze verantwoordelijk groeien in het element.

Ik heb je feedback ook verwerkt met het maken van een tweede breakpoint in de footer bij een tablet-esque schermgrootte

nadiachaja commented:
Ik vind de logo's best wel groot op desktop. Ik zou ook alles wat boven zit wat meer ruimte boven geven zit nu dicht op de rand. 
Ik zou naar het design kijken hoe dat eruit ziet. 
<img width="806" height="248" alt="Scherm­afbeelding 2025-10-22 om 19 38 43" src="https://github.com/user-attachments/assets/0b001449-3199-4c43-b741-bedc6162a24a" />

Sidopjescherm committed:
#309 Logo fix

@nadiachaja Hierbij een verkleinde klamp. Het lukt mij alleen niet om tussen 767 en 855 het niet te laten squishen zelfs met clamp. Wel is de padding gefixt

nadiachaja commented:
De padding ziet er goed. Je kan misschien de logo op een bepaald moment onder elkaar zetten. Als dat niet mooi is kan je misschien de max-width veranderen of alleen width gebruiken.

Sidopjescherm committed:
#309 Ademruimte logo's

Ik heb de display naar grid gemaakt zodat het onder elkaar staat en een gap van 2rem zodat ze wel genoeg afstand van elkaar hebben

nadiachaja commented:
De logo's zien er goed uit.  Ik zie wel dat de tekst van de navigatie iets meer naar het midden kan. Dat ziet er beter uit. 
<img width="533" height="363" alt="Scherm­afbeelding 2025-10-22 om 20 46 45" src="https://github.com/user-attachments/assets/e0657cff-7a64-4575-bec6-7fecb8db49ea" />

Sidopjescherm committed:
#309 Centreren tekst

heb het aangepast. het ziet er inderdaad beter uit zo. Had er helemaal overheen gekeken, excuus
